docs/modules/ROOT/pages/install.adoc

```diff
@@ -0,0 +1,69 @@
+= Install
+
+You can install the extension from your browser's web-store, or a local copy of the extension app.
+
+== Install from web-store
+
+To install the Extension from your browser's web-store:
+
+. Follow the image links below to install the extension direct from your browser's web-store or search your browser's web-store for the Asciidoctor.js Live Preview extension.
++
+[%autowidth,cols=2*a]
+|===
+|[caption=]
+.Chrome
+image::chrome-webstore-badge.png["Image link to the Chrome, Chromium or Brave web-store",212,64,link={url-chrome-webstore}]
+
+|[caption=]
+.Firefox
+image::firefox-webstore-badge.svg[Image link to the Firefox web-store,184,64,link={url-mozilla-addon}]
+
+|[caption=]
+.Opera
+image::opera-webstore-badge.png[Image link to the Opera web-store,227,64,link={url-opera-addon}]
+
+|[caption=]
+.Edge
+image::edge-webstore-badge.png[Image link to the Microsoft Edge web-store,177,64,link={url-edge-webstore}]
+|===
+
+. In order to preview local files on {chromium-browser-list} browsers,
+you will need to grant filesystem access to the extension:
+
+.. Type the URL `chrome://extensions` into the address bar and press kbd:[Enter]
+.. Type `asciidoctor` into the extension search bar then when the {project-name} block is visible, click its btn:[Details] button... Check `Allow access to file URLs`
+
+. You have now installed the extension, and can open local or remote *.adoc*, *.ad*, *.asc*, *.asciidoc* files in your browser
++
+Enjoy!
+
+[NOTE]
+====
+If you are using a Chromium browser, and want to view local files, remember to enable `Allow access to file URLs` above.
+====
+
+NOTE: There are known issues with Firefox. See xref:firefox-known-issues.adoc[].
+
+== Local installations
+
+Local installations are for users and developers when:
+
+* Your machine does not have access to the internet or extension stores
+* You are testing an unreleased (locally built) version
+
+Loading your local copy of the extension on {chromium-browser-list} or Firefox is super easy!
```

Comment:
I believe we shouldn't use "super easy" in documentation. It can be frustrating if, for some reasons, it's not super easy for you 😄

Reply:
I do agree. It irks me reading documentation that says "simply...." as people approach it from both foundation and advanced knowledge levels.

Reply:
Resolved by https://github.com/asciidoctor/asciidoctor-browser-extension/pull/649/commits/9ae1bfab173da12a0a5e094bee05dab3c6ee4cb6